themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -151,4 +151,19 @@
   </div>
 </body>
 </html>
+
+<script>
```

Comment:
Can we move this script before the closing tag of the HTML document? I don't think this is valid HTML as-is.

Reply:
@vplemyannik perhaps this is the reason the tests are failing, but I am not sure.

Reply:
Yeap, i have moved script section before HTML closing tag